README.md

```diff
@@ -9,9 +9,6 @@
 * install dependences: `npm i`
 
 ## Development
-To start the app, simply run `npm run start-webapp`
-The app should run at `localhost:8080`
-
 To start the server, simply run `npm run start-server`. 
```

Comment:
This can be renamed to `npm run start` as `server` is now implied,

Reply:
Updated!